### DISCUSSION TITLE
How to use only custom props on trackEvent? I mean only my own params instdead of (event, category, option .....etc)

### DISCUSSION THREAD

**Discussion by antonpatotski** [gtm-support/vue-gtm#396] [Q&A] [answered]
Hi! Could please somebody help with this problem. We have setted up google analitics, and we don't need default props. How to avoid send them?

**Comment by Shinigami92** [comment]
Did you try `push`?

**Reply by mrc-bsllt** [reply, marked as answer]
> Did you try `push`?

This is how it is done, right?

```javascript
const gtm = useGtm();

gtm?.push({ event: "me-event", customProp: "my custom prop" });
```

**Reply by Shinigami92** [reply]
Yes, but also keep in mind: `The event will only be send if the script runs in browser context, dataLayer exists, and the plugin is enabled.`
